src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
 
+        // This represents how much EC to consume per executed instruction.
+        // This would be the "variable" component of the processor's power.
+        // TODO: This value needs to be changed to a non-zero number and balanced.
```

Comment:
In fact, now that I think of it, we really should just do that during testing.  Let's leave this as hardcoded to `0F` in the code now and forever, and instead play with values by fiddling with the part config files - that way we're doing the same thing the end user can, by "pulling on" the same handle they have available to tweak, and therefore testing even more of the whole chain of events.

Just put a comment in here in the code mentioning that the only reason this field is zero in the code is because the Part.cfg files override it, and it's not *really* zero when the game is running.  (otherwise reading the code may give people the wrong idea down the road).